testing/trino-testing/src/main/java/io/trino/testing/TestingConnectorBehavior.java

```diff
@@ -22,7 +22,6 @@
     SUPPORTS_PREDICATE_PUSHDOWN,
```

Comment:
@findepi Please take a look at the "grouping" in this class since this is what's now being followed everywhere.

Today it looks like:

```java
// Predicate and expression pushdowns
SUPPORTS_PREDICATE_PUSHDOWN,
SUPPORTS_PREDICATE_PUSHDOWN_WITH_VARCHAR_EQUALITY,
SUPPORTS_PREDICATE_PUSHDOWN_WITH_VARCHAR_INEQUALITY,
SUPPORTS_PREDICATE_EXPRESSION_PUSHDOWN,
SUPPORTS_PREDICATE_EXPRESSION_PUSHDOWN_WITH_LIKE,

SUPPORTS_DYNAMIC_FILTER_PUSHDOWN,

SUPPORTS_LIMIT_PUSHDOWN,

// TopN pushdowns
SUPPORTS_TOPN_PUSHDOWN,
SUPPORTS_TOPN_PUSHDOWN_WITH_VARCHAR,

// Aggregation pushdowns
SUPPORTS_AGGREGATION_PUSHDOWN,
SUPPORTS_AGGREGATION_PUSHDOWN_STDDEV,
SUPPORTS_AGGREGATION_PUSHDOWN_VARIANCE,
SUPPORTS_AGGREGATION_PUSHDOWN_COVARIANCE,
SUPPORTS_AGGREGATION_PUSHDOWN_CORRELATION,
SUPPORTS_AGGREGATION_PUSHDOWN_REGRESSION,
SUPPORTS_AGGREGATION_PUSHDOWN_COUNT_DISTINCT,

// Join pushdowns
SUPPORTS_JOIN_PUSHDOWN,
SUPPORTS_JOIN_PUSHDOWN_WITH_FULL_JOIN,
SUPPORTS_JOIN_PUSHDOWN_WITH_DISTINCT_FROM,
SUPPORTS_JOIN_PUSHDOWN_WITH_VARCHAR_EQUALITY,
SUPPORTS_JOIN_PUSHDOWN_WITH_VARCHAR_INEQUALITY,

// Schema DDL
SUPPORTS_CREATE_SCHEMA,
SUPPORTS_RENAME_SCHEMA,

// Table DDL
SUPPORTS_CREATE_TABLE,
SUPPORTS_CREATE_TABLE_WITH_DATA,
SUPPORTS_CREATE_TABLE_WITH_TABLE_COMMENT,
SUPPORTS_CREATE_TABLE_WITH_COLUMN_COMMENT,
SUPPORTS_RENAME_TABLE,
SUPPORTS_RENAME_TABLE_ACROSS_SCHEMAS,

// Column DDL
SUPPORTS_ADD_COLUMN,
SUPPORTS_ADD_COLUMN_WITH_COMMENT,
SUPPORTS_DROP_COLUMN,
SUPPORTS_RENAME_COLUMN,

// COMMENT ON DDL
// Should this be merged with Table DDL, Column DDL and SUPPORTS_CREATE_VIEW? @findepi
SUPPORTS_COMMENT_ON_TABLE,
SUPPORTS_COMMENT_ON_VIEW,
SUPPORTS_COMMENT_ON_COLUMN,

SUPPORTS_CREATE_VIEW,

// MV DDL
SUPPORTS_CREATE_MATERIALIZED_VIEW,
SUPPORTS_RENAME_MATERIALIZED_VIEW,
SUPPORTS_RENAME_MATERIALIZED_VIEW_ACROSS_SCHEMAS,

// All these are allowed to be merged into single group if multiple of them have same value (DML group)
// Should I group these together within file as well? @findepi 
// DML START //
SUPPORTS_INSERT,
SUPPORTS_NOT_NULL_CONSTRAINT,

SUPPORTS_DELETE,
SUPPORTS_ROW_LEVEL_DELETE,

SUPPORTS_UPDATE,

SUPPORTS_MERGE,

SUPPORTS_TRUNCATE,
// DML END //

// Data type support
SUPPORTS_ARRAY,
SUPPORTS_ROW_TYPE,
SUPPORTS_NEGATIVE_DATE,

SUPPORTS_CANCELLATION,

SUPPORTS_MULTI_STATEMENT_WRITES
```